integration/env/aws.go

```diff
@@ -1,5 +1,3 @@
-// +build k8srequired
-
```

Comment:
Why removing this? What is the rule? I would like to keep this usefully consistent.

Reply:
I just copied it from e2esetup.

Reply:
Restored.